src/components/uielements/LanguageMenu.astro

```diff
@@ -0,0 +1,74 @@
+---
+import iconWorld from "@assets/icons/world.svg";
+import { languages } from "@src/locales/consts";
+import { getLocale } from "i18n:astro";
+
+const basePath = import.meta.env.PUBLIC_ASTRO_BASE_PATH;
```

Comment:
PUBLIC_ASTRO_BASE_PATH isn't used in other places anymore it seems... 
![image](https://github.com/user-attachments/assets/123a5693-848e-45dc-ae8e-5b799ea7169b)

Not sure if this is correct but now the languagemenu sends me to a 404 page: 
![image](https://github.com/user-attachments/assets/c6486cb1-2cac-4502-9dcd-e9248e2595b6)


Might be fixed in some other branch..